build/pom.xml

```diff
@@ -42,6 +42,12 @@
     <name>WildFly: Build</name>
     <packaging>pom</packaging>
 
+    <properties>
+        <!-- Galleon seems to understand properties with these exact names when it does expression resolution. -->
```

Comment:
"seems" or "is expected to" :)

Reply:
s/seems to understand/understands/g

I'll change that.